.circleci/test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+
+# load cache from build job
+docker load < ~/docker-cache/image.tar
+
+# download Reaction to test the build
+git clone https://github.com/reactioncommerce/reaction.git /tmp/reaction
+
+# make sure we have a compatible Dockerfile in the Reaction dir
+cp .circleci/test.Dockerfile /tmp/reaction/Dockerfile
+
+cd /tmp/reaction
+
+# build new image
+docker build \
+  --build-arg TOOL_NODE_FLAGS="--max-old-space-size=4096" \
+  -t reaction-test .
+
+# run the container and wait for it to boot
+docker-compose -f .circleci/docker-compose.yml up -d
+sleep 30
+
+# use curl to ensure the app returns 200's
+docker exec reaction bash -c "apt-get update && apt-get install -y curl && \
+  curl --retry 10 --retry-delay 10 -v http://localhost:3000"
```

Comment:
This test is basically checking downstream usage of this Docker image by Reaction which isn't a concern of this project, IMO. It may be better to keep the tests to things expected in this image, like presence of Meteor. Would make the tests simpler and reduce cross project dependency.

Reply:
That's technically true, but that test was only added to ensure the _only_ thing this image is intended to be used for actually works.  You can remove it if you want, but I don't see any harm in testing that the final built image works with the current stable release of Reaction.

Reply:
Reasoning about this again today, I think since we're maintaining this as a separate repo, this test is useful to help prevent shipping a breaking version of the base. 

Agreed, we can consider using other means to check that the image contains expected dependencies, but considering the frequency of changes that this base gets, this is good.